source/deployment/guest-accounts.rst

```diff
@@ -3,63 +3,105 @@
+The Guest Account feature controls access to channels and other members within the organization.
 
-An organization may use guest accounts to work on a website project with resources from external design firm. Guest Accounts could also be used to collaborate with customers on a support issue.
+For example, an organization may use Guest Accounts to work on a website project with resources from an external design firm. Guest Accounts could also be used to collaborate with customers on a support issue.
 
-Guests can: 
+Guests can:
 
 - Pin messages to channels
 - Use slash commands (excluding restricted commands such as invite members, rename channels, change headers, etc)
 - Favorite channels
 - Mute channels
 - Update their account settings
 
-Guests cannot: 
+Guests cannot:
 
 - Discover public channels
 - Join open teams
 - Create direct messages or group messages with members who aren’t within the same channel
+- Invite people
 
 
 Enabling Guest Accounts
 ------------------------
 
 1. Navigate to **System Console > Authentication > Guest Access**.
 2. Set **Enable Guest Access** to ``True``.
-3. (Optional) **Whitelist domains that are acceptable for Guest Access**.  
+3. (Optional) **Whitelist domains that are acceptable for Guest Access**.
  - This allows the System Administrators to set a list of approved guest domains. If you have team domain restrictions, you will also need to add your guest domain to **Team Settings > Allow only users with a specific email domain to join this team**.
 4. (Optional) **Enforce Multi-Factor Authentication (MFA) for your Guests**.
- - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users. 
+ - If you are enforcing MFA for your users, you can optionally choose to enforce MFA for your guest users.
 
 Guest Authentication
------------------------------
-Guests can access the Mattermost server via email authentication. Please ensure you enable email invites from **System Console > Signup > Enable Email Invitations**.   
+---------------------
 
-SAML 2.0 and AD/LDAP authentication will be provided in a future release.
+Guests can access the Mattermost server via email invitation, and be authenticated using AD/LDAP or SAML 2.0.
+Before you proceed, ensure that the authentication method you wish to use is correctly configured on your server and enabled in Mattermost.
+For configuration steps and technical documentation, see `Active Directory/LDAP Setup <https://docs.mattermost.com/deployment/sso-ldap.html>`_
+and `SAML Single-Sign-On <https://docs.mattermost.com/deployment/sso-saml.html>`_.
 
-Inviting Guests to the Mattermost Server
+Converting a member user to a guest will not change the channels they are in; however they will be restricted from discovering additional channels and unable to direct message / group message users outside of the channels they are in. They can be added to channels by Team and System Admins.
```

Comment:
> They can be added to channels by Team and System Admin. 

System Admins by default and only those with the "Invite Guest" Permission.  All other roles are defaulted to false for invite guests, so it really depends on their permissions schemes.